From 5691f292bdf793e018d0d51aabec98c395c50593 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Tue, 6 Dec 2016 18:06:47 -0800
Subject: [PATCH] Fix infinite loop on exit

---
 libretro/libretro.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libretro/libretro.cpp b/libretro/libretro.cpp
index 89552dc..e3505e1 100644
--- a/libretro/libretro.cpp
+++ b/libretro/libretro.cpp
@@ -593,10 +593,10 @@ static void retro_wrap_emulator(void)
    retro_start_emulator();
 
    // Exit comes from DOSBox, tell the frontend
-   /*if(DOSBOXwantsExit)
+   if(DOSBOXwantsExit)
    {
       environ_cb(RETRO_ENVIRONMENT_SHUTDOWN, 0);
-   }*/
+   }
 
    // Were done here
    co_switch(mainThread);
-- 
2.7.4

