From 4ccebfd0cea835f7720a9e8eb1addc977565edd5 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Tue, 17 Jan 2023 17:50:52 -0800
Subject: [PATCH] Build with -std=gnu++11

---
 Makefile.libretro | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/Makefile.libretro b/Makefile.libretro
index ac63861..baf1dec 100644
--- a/Makefile.libretro
+++ b/Makefile.libretro
@@ -24,6 +24,10 @@ SOURCES_CXX :=
 
 include Makefile.common
 
+# Needed for building with GCC 11
+# See https://github.com/libretro/dosbox-libretro/issues/137
+CXXFLAGS += -std=gnu++11
+
 # try to guess the dynarec based on the host system, unreliable
 ifeq ($(platform),win)
 	ifneq ($(findstring MINGW32,$(UNAME)),)
@@ -189,7 +193,7 @@ else ifeq ($(platform), libnx)
 	TARGET := $(TARGET_NAME)_libretro_$(platform).a
 	CFLAGS += -g -O2 -fPIE -I$(LIBNX)/include/ -ffunction-sections -fdata-sections -ftls-model=local-exec -Wl,--allow-multiple-definition -specs=$(LIBNX)/switch.specs
 	CFLAGS += $(INCLUDE) -D__SWITCH__ -DHAVE_LIBNX
-	CXXFLAGS += $(ASFLAGS) $(CFLAGS) -std=gnu++11
+	CXXFLAGS += $(ASFLAGS) $(CFLAGS)
 	CFLAGS += -std=gnu11
 	WITH_DYNAREC =
 	STATIC_LINKING = 1
-- 
2.34.1

